extensions/amp-date-display/0.1/amp-date-display.js

```diff
@@ -96,49 +96,42 @@ export class AmpDateDisplay extends AMP.BaseElement {
     /** @private {string} */
     this.locale_ = '';
 
-    /** @const {!../../../src/service/template-impl.Templates} */
+    /** @private @const {!../../../src/service/template-impl.Templates} */
     this.templates_ = Services.templatesFor(this.win);
+
+    /** @private {?Element} */
+    this.container_ = null;
   }
 
   /** @override */
   buildCallback() {
     if (!isExperimentOn(this.win, 'amp-date-display')) {
-      user().warn(TAG, 'Experiment %s is not turned on.', TAG);
+      user().error(TAG, 'Experiment %s is not turned on.', TAG);
```

Comment:
should be a `userAssert` but its going away anyway.

Reply:
👍